modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/tests/[UI]DefineAPISchemasWithinAnAPIGroup/DeleteAPIApplicationSchema.testcase

```diff
@@ -0,0 +1,173 @@
+	property portal.release = "true";
+	property portal.upstream = "true";
+	property testray.component.names = "Object";
+	property testray.main.component.name = "Headless Builder";
+
+	setUp {
+		TestCase.setUpPortalInstance();
+
+		User.firstLoginUI();
+
+		task ("Given API Application with Schema created") {
+			ApplicationAPI.createAPIApplication(
+				baseURL = "my-app",
+				mainObjectDefinitionErc = "L_API_APPLICATION",
+				relatedSchema = "true",
+				schemaName = "testSchema",
+				status = "published",
+				title = "My-app");
+
+			ApplicationAPI.setUpGlobalAPIApplicationId();
+		}
+	}
+
+	tearDown {
+		ApplicationAPI.deleteAllAPIApplication();
+
+		var testLiferayVirtualInstance = PropsUtil.get("test.liferay.virtual.instance");
+
+		if (${testLiferayVirtualInstance} == "true") {
+			PortalInstances.tearDownCP();
+		}
+	}
+
+	@priority = 3
+	test CanCancelAPISchemaDeletion {
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given delete API Schema") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "My-app");
+
+			Click(
+				key_title = "testSchema",
+				locator1 = "APIBuilder#DROPDOWN_MENU");
+
+			Click(locator1 = "APIBuilder#DELETE_IN_DROPDOWN_MENU");
+		}
+
+		task ("When I click cancel button") {
+			Click(locator1 = "Button#CANCEL");
+		}
+
+		task ("Then I can see the API Schema") {
+			AssertElementPresent(
+				key_name = "testSchema",
+				locator1 = "ObjectAdmin#TABLE_LIST_TITLE");
+		}
+	}
+
+	@priority = 3
+	test CanDeleteAPISchema {
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given goto dropdown menu and click delete") {
+			APIBuilderUI.switchToRelatedEntryInEditApplication(
+				entity = "Schemas",
+				title = "My-app");
+
+			Click(
+				key_title = "testSchema",
+				locator1 = "APIBuilder#DROPDOWN_MENU");
+
+			Click(locator1 = "APIBuilder#DELETE_IN_DROPDOWN_MENU");
+		}
+
+		task ("When I click delete button") {
+			Click(locator1 = "APIBuilder#MODAL_DELETE_BUTTON");
+		}
+
+		task ("Then the API Schema has been deleted in the UI") {
+			AssertElementNotPresent(
+				key_name = "testSchema",
+				locator1 = "ObjectAdmin#TABLE_LIST_TITLE");
+		}
+	}
+
+	@priority = 4
+	test CannotSeeSchemaInAPIExplorerAfterDelete {
+		property portal.acceptance = "true";
+		property test.liferay.virtual.instance = "false";
+		property test.run.type = "single";
+
+		task ("And Given create an API Endpoint in the Application") {
+			EndpointAPI.createAPIEndpoint(
+				apiApplicationId = ${staticApplicationId},
+				name = "myEndpoint",
+				path = "/myEndpoint");
+		}
+
+		task ("And Given navigate to API Explorer's created API Application") {
+			APIExplorer.navigateToOpenAPI(customObjectPlural = "my-app");
+		}
+
+		task ("And GIven I can see the API Schema and API Endpoint") {
```

Comment:
I think we may already have related test for this `And GIven I can see the API Schema and API Endpoint`. Could you check and once we confirm it exists then we can remove from here, thanks

Reply:
There have been tests that included this assert and if we delete this step, the front step also needs to delete